osu.Game.Rulesets.Osu/Mods/OsuModAutopilot.cs

```diff
@@ -30,16 +29,20 @@ public class OsuModAutopilot : Mod, IApplicableFailOverride, IUpdatableByPlayfie
 
         public bool RestartOnFail => false;
 
-        private OsuInputManager inputManager;
+        private OsuInputManager? inputManager;
 
-        private IFrameStableClock gameplayClock;
+        private IFrameStableClock? gameplayClock;
 
-        private List<OsuReplayFrame> replayFrames;
+        private List<OsuReplayFrame>? replayFrames;
```

Comment:
not the first time this has come up but i still think these should be non-nullable, similar to DI required properties.